src/styleguide/_patterns/01-atoms/06-buttons/08-toggle-menu.mustache

```diff
@@ -1,13 +1,4 @@
-<p class="is-light">
-  <button type="button" class="grav-c-toggle-menu" aria-pressed="{{control.pressed}}" aria-label="{{control.label}}">
-   <span class="grav-c-toggle-menu__icon"></span>
-  </button>
-</p>
-
-<h4>Dark background</h4>
 
-<p class="sg-dark-background">
```

Comment:
this class is not needed anymore, you need to remove it.

Reply:
done.